bin/dbsv-update.php

```diff
@@ -1175,6 +1175,49 @@ function dbv_160()
     );
 }
 
+function dbv_161()
+{
+    global $opt;
+
+    // add nuts codes for Bulgaria, Kroatia, Romania and Turkey
+    system(
+        'cat ' . __DIR__. '/../sql/static-data/nuts_codes.sql |' .
+        ' mysql -h' . $opt['db']['servername'] . ' -u' . $opt['db']['username'] .
+        ' --password=' . $opt['db']['password'] . ' ' . $opt['db']['placeholder']['db']
+    );
+
+    // recalculate cache locations for these countries
+
+    // temporarily get rid of the update trigger so that we can change last_modified
+    update_triggers();
+    sql(
+        "UPDATE `cache_location`
+         SET `last_modified` = '2000-01-01 00:00:00'
+         WHERE `code1` IN ('BG', 'HR', 'RO', 'TR')"
+    );
+
+    // update the cache locations while trigger is disabled, so that the
+    // modification dates of caches will not be touched
+    function checkJob()
+    {
+    }
```

Comment:
what does it check?

Reply:
See cache_location_class.php. It's a dummy function needed to include the cronjob module here. But ... well, we may save a line of code bei using it to run the job. :-)

Reply:
one more reason why this should be improved ;-)

Reply:
done